autoscale/src/main/resources/logback.xml

```diff
@@ -35,7 +35,7 @@
         <appender-ref ref="STDOUT"/>
     </root>
 
-    <logger name="com.sequenceiq" level="${PERISCOPE_LOG_LEVEL:-INFO}" additivity="false">
+    <logger name="com.sequenceiq" level="${PERISCOPE_LOG_LEVEL:-DEBUG}" additivity="false">
```

Comment:
I see this one makes it the same as other services.

Reply:
Yes, I may still roll back this change. AutoScale DEBUG may be a little too chatty.